content/en/docs/ops/deployment/requirements/index.md

```diff
@@ -65,22 +65,19 @@ The following ports and protocols are used by Istio.
 | 15000 | TCP | Envoy | Envoy admin port (commands/diagnostics) |
 | 15001 | TCP | Envoy | Envoy Outbound |
 | 15006 | TCP | Envoy | Envoy Inbound |
-| 15020 | HTTP | Envoy | Health checks |
-| 15090 | HTTP | Envoy | Prometheus telemetry |
+| 15021 | HTTP | Envoy | Health checks |
+| 15020 | HTTP | Envoy | Istio agent Prometheus telemetry |
```

Comment:
Either the order is wrong, or the ports are mixed up. Just a double check.

Reply:
the ports are correct. The ordering isn't numberical its sort of random currently, but probably makes sense to swap these